### DISCUSSION TITLE
I18N: Support for Right-to-Left Languages

### DISCUSSION THREAD

**Discussion by ebrahim** [modoboa/modoboa#508] [Ideas]
Special care is needed for right-to-left languages, both for UI layout and for content. Supporting right-to-left languages such as Persian and Arabic would be nice.

**Comment by tonioo** [comment]
Hi, as I'm not familiar with right-to-left languages, do you know resources that would help to start ? Or maybe people ?

**Comment by ebrahim** [comment]
I'm glad you care about the issue. :)

Found these by searching the net that may help in getting you introduced:
- http://www.i18nguy.com/MiddleEastUI.html
- http://docs.kendoui.com/tutorials/accessibility/supporting-rtl-languages
- http://dotancohen.com/howto/rtl_right_to_left.html
- http://viget.com/flourish/designing-for-rtl-languages-the-basics
- http://www.w3.org/TR/html401/struct/dirlang.html
